app/front/scripts/directives/visualizations/piechart/template.html

```diff
@@ -8,6 +8,7 @@
     state="state"
     format-value="formatValue"
     messages="messages"
-    endpoint="{{ params.babbageApiUrl }}">
+    endpoint="{{ params.babbageApiUrl }}"
+    max-slices="5">
```

Comment:
Ah, nm, found it.
Better to update the babbage.ui version dependency to make sure it uses the version that defines this parameter.